client/scss/components/_body_content.scss

```diff
@@ -45,3 +45,54 @@
     }
   }
 }
+
+$large-width: (5/7 * 100)+'%';
```

Comment:
I've just removed the other variables, they were from the [old `--full-width`](https://github.com/wellcometrust/wellcomecollection.org/blob/7d89c92f09fd3def7453f315002d6667c74ff152/client/scss/components/_figure.scss#L76).
Just applying the [new `--full-width`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/client/scss/components/_figure.scss#L78) doesn't work, so I'd like to approach that in another PR, hence the TODO.